tests/test_check_cloudformation.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import patch
+
+import pytest
+from click.testing import CliRunner
+
+import commands
+from commands.check_cloudformation import check_cloudformation as check_cloudformation_command, valid_checks
+
+
+class TestCheckCloudformationCommand:
```

Comment:
What's the rationale for using a class here? We don't do this in other test modules in copilot-tools or the pipelines repo

Reply:
I took my cue from https://github.com/uktrade/copilot-tools/blob/main/tests/test_copilot_cli.py#L11

I'll lose the class though actually as I think it's better to break things out into more separate files to keep it all manageable.

Reply:
It makes sense to just have a test_{file} for each {file} in which case it's helpful to group related tests under classes.  The class is just a grouping mechanism.